PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/RuleConfigLoader.java

```diff
@@ -132,6 +132,8 @@ private RuleConfig getRuleConfig(JsonReader reader) throws IOException {
         List<String> servicesToCheck = new ArrayList<>();
         Map<String, String> antiPatternMessageMap = new HashMap<>();
         List<String> listedItemsToCheck = new ArrayList<>();
+        Map<String, String> recommendationTextMap = new HashMap<>();
+        Map<String, String> recommendationLinkMap = new HashMap<>();
```

Comment:
Can these all now be updated to a better data structure?

Reply:
discussed offline